code/datums/communications.dm

```diff
@@ -132,23 +134,24 @@ datum/radio_frequency
 				reusable_signals |= signal
 			LAGCHECK(LAG_MED)
 
+		// check_for_jammer(obj/source)
```

Comment:
oh shit. should just remove this. no need for it anymore. I'll do it when I have time